doc/content/gettingstarted/events.md

```diff
@@ -0,0 +1,208 @@
+---
+title: "ADV - Events"
```

Comment:
`Advanced: Events` ?

Reply:
Yeah, I just feel this is a bit too complex and unnecessary in the `getting started`, which should be just about setting the stack until having "standard" traffic arrives to your platform. 
This would be neater if we instead put it as a reference or in a `Recommended readings` section.

Reply:
Fine, then move it to an advanced section or something